doc/examples/parameter_server/sync_parameter_server.py

```diff
@@ -46,15 +46,16 @@ def compute_gradients(self, weights):
 if __name__ == "__main__":
     args = parser.parse_args()
 
-    ray.init(redis_address=args.redis_address)
+    ray.init(address=args.address)
 
     # Create a parameter server.
     net = model.SimpleCNN()
     ps = ParameterServer.remote(1e-4 * args.num_workers)
 
     # Create workers.
-    workers = [Worker.remote(worker_index)
-               for worker_index in range(args.num_workers)]
+    workers = [
+        Worker.remote(worker_index) for worker_index in range(args.num_workers)
+    ]
```

Comment:
I think the issue here is not that I have the wrong version of yapf (I'm using 0.23.0) but rather that these files haven't been touched in a while?